src/apps/bm_devkit/bmdk_common/app_main.cpp

```diff
@@ -138,9 +139,9 @@ SerialHandle_t usbPcap = {
 };
 
 // TODO - make a getter API for these
-cfg::Configuration *userConfigurationPartition = NULL;
-cfg::Configuration *systemConfigurationPartition = NULL;
-cfg::Configuration *hardwareConfigurationPartition = NULL;
+static NvmPartition *userConfigurationPartition = NULL;
+static NvmPartition *systemConfigurationPartition = NULL;
+static NvmPartition *hardwareConfigurationPartition = NULL;
```

Comment:
Was able to get rid of these global variables. Now they are just local to this file for the callback functions.

Reply:
Sadly these have to be globals now again with the config wrapper updates